src/test/scala/com/atomist/project/archive/RugArchiveReaderTest.scala

```diff
@@ -0,0 +1,135 @@
+package com.atomist.project.archive
```

Comment:
We need these tests right now, especially during the refactoring work I've been doing.